devops/services/nginx/setup_nginx_service.yml

```diff
@@ -7,8 +7,7 @@
     image_tag: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:image_tag') }}"
     replicas: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:replicas') }}"
     domain_name: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:domain_name') }}"
-    fullchain: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:fullchain') }}"
-    privkey: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:privkey') }}"
+    pull_certificates: "{{ lookup('community.hashi_vault.hashi_vault', 'secret=secret/data/prod/nginx:pullcert') }}"
```

Comment:
Hey @Manas23601 let's use the same variable name in vault as being used in ansible.